CHANGELOG.md

```diff
@@ -4,8 +4,82 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 
-## [Unreleased]
+## [0.6.0] - 2021-01-XX
+### Compatibility Changes:
+- support for CUDA 11, 11.1, and 11.2 #1076 #1086 #1147 #1231
+- support for CMake 3.18.0 and 3.19.0 #1087 #1217
+- set minimal HIP version to 3.5 #1110
+- remove CMake HIP module shipped with alpaka #1189
+- set HIP-clang as default compiler for HIP #1113
+- support for NVCC + VS 2019 #1121
+- remove support for CUDA 11.0 with MSVC 2019 #1227
```

Comment:
That should be together with the first line stating that we added support for CUDA 11. E.g.:
`support for CUDA 11 (except MSVC 2019), 11.1, and 11.2 #1076 #1086 #1147 #1231 #1227`